app/Policies/V2/AuditStatus/AuditStatusPolicy.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Policies\V2\AuditStatus;
+
+use App\Models\User;
+use App\Models\V2\AuditStatus\AuditStatus;
+use App\Policies\Policy;
+
+class AuditStatusPolicy extends Policy
+{
+    public function uploadFiles(?User $user, ?AuditStatus $auditStatus): bool
+    {
+        return true;
```

Comment:
There isn't much point in a policy that is always true. Is this really valid? If it is, I think simply removing this policy and the the policy check in the controller that uses it would be fine.